SecretSanta/src/SecretSanta.Api/Controllers/UsersController.cs

```diff
@@ -43,17 +61,35 @@ public ActionResult Delete(int id)
         }
 
         [HttpPost]
-        public ActionResult<User?> Post([FromBody] User? user)
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(User), StatusCodes.Status200OK)]
+        public ActionResult<DtoUser?> Post([FromBody] DtoUser? dtoUser)
         {
-            if (user is null)
+            if (dtoUser is null)
             {
                 return BadRequest();
             }
-            return Repository.Create(user);
+            int id;
+            if(Repository.List().Count == 0){
+                id = 0;
+            }
```

Comment:
nit: We typically place braces on their own lines.
```suggestion
            if(Repository.List().Count == 0)
            {
                id = 0;
            }
```